packages/scope-sdk/src/constants/Mints.ts

```diff
@@ -0,0 +1,64 @@
+  { token: 'cETH', mint: 'FbKvdbx5h6F86h1pZuEqv7FxwmsVhJ88cDuSqHvLm6Xf' },
+  { token: 'cBTC', mint: 'Gqu3TFmJXfnfSX84kqbZ5u9JjSBVoesaHjfTsaPjRSnZ' },
+  { token: 'cMSOL', mint: '3JFC4cB56Er45nWVe29Bhnn5GnwQzSmHVf6eUq9ac91h' },
+  { token: 'wstETH', mint: 'ZScHuTtqZukUrtZS43teTKGs2VqkKL8k4QCouR2n6Uo' },
+  { token: 'LDO', mint: 'HZRCwxP2Vq9PCpPXooayhJ2bxTpo5xfpQrwB1svh332p' },
+  { token: 'USDC', mint: 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v' },
+  { token: 'cUSDC', mint: '993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk' },
+  { token: 'USDT', mint: 'Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB' },
+  { token: 'ORCA', mint: 'orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE' },
+  { token: 'MNDE', mint: 'MNDEFzGvMt87ueuHvVU9VcTqsAP5b3fTGPsHuuPA5ey' },
+  { token: 'HBB', mint: 'HBB111SCo9jkCejsZfz8Ec8nH7T6THF8KEKSnvwT6XK6' },
+  { token: 'cORCA', mint: 'E9LAZYxBVhJr9Cdfi9Tn4GSiJHDWSZDsew5tfgJja6Cu' },
+  { token: 'cSLND', mint: 'D3Cu5urZJhkKyNZQQq2ne6xSfzbXLU4RrywVErMA2vf8' },
+  { token: 'cSRM', mint: '4CxGuD2NMr6zM8f18gr6kRhgd748pnmkAhkY1YJtkup1' },
+  { token: 'cRAY', mint: '2d95ZC8L5XP6xCnaKx8D5U5eX6rKbboBBAwuBLxaFmmJ' },
+  { token: 'cFTT', mint: 'DiMx1n2dJmxqFtENRPhYWsqi8Mhg2p39MpTzsm6phzMP' },
+  { token: 'cSTSOL', mint: 'QQ6WK86aUCBvNPkGeYBKikk15sUg6aMUEi5PTL6eB4i' },
+  { token: 'SLND', mint: 'SLNDpmoWTVADgEdndyvWzroNL7zSi1dF9PC3xHGtPwp' },
+  { token: 'DAI', mint: 'EjmyN6qEC1Tf1JxiG1ae7UTJhUxSwk1TCWNWqxWV4J6o' },
+  { token: 'JSOL', mint: '7Q2afV64in6N6SeZsAAB81TJzwDoD6zpqmHkzi9Dcavn' },
+  { token: 'USH', mint: '9iLH8T7zoWhY7sBmj1WK9ENbWdS1nL8n9wAxaeRitTa6' },
+  { token: 'UXD', mint: '7kbnvuGBxxj8AG9qp8Scn56muWGaRaFqxg1FsRp3PaFT' },
+  { token: 'HDG', mint: '5PmpMzWjraf3kSsGEKtqdUsCoLhptg4yriZ17LKKdBBy' },
+  { token: 'DUST', mint: 'DUSTawucrTsGU8hcqRdHDCbuYhCPADMLM2VcCb8VnFnQ' },
+  { token: 'kUSDHUSDCOrca', mint: '5BmZgW7dk1kximGfn7MPvDigp3yRmgT64jS9Skdq4nPY' },
+  { token: 'kSOLSTSOLOrca', mint: 'C7RpHABBJjfkmizgnR2MRbpfEF454Rb18XC6RmBC8cGg' },
+  { token: 'kUSDCUSDTOrca', mint: 'CHdEG9MWBVT1aveiFSMMUDWvo3D42EBdHxT9q9BNZWLu' },
+  { token: 'kUSHUSDCOrca', mint: 'GnsoaSc8oi53oC2Nq7F4w4ke8rSy2HgNeuhSSVxnVdXJ' },
+  { token: 'USDR', mint: 'USDrbBQwQbQ2oWHUPfA8QBHcyVxKUq1xHyXsSLKdUq2' },
+  { token: 'RATIO', mint: 'ratioMVg27rSZbSvBopUvsdrGUzeALUfFma61mpxc8J' },
+  { token: 'UXP', mint: 'UXPhBoR3qG4UCiGNJfV7MqhHyFqKN68g45GoYvAeL2M' },
+  { token: 'kUXDUSDCOrca', mint: '4G9USgnbg6fDTQ5AUfpCjM89zqbzWj32xfqvsaAu66DM' },
+  { token: 'JITOSOL', mint: 'J1toso1uCk3RLmjorhTtrVwY9HJ7X8V9yYac6Y7kGCPn' },
+  { token: 'BONK', mint: 'DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263' },
+  { token: 'SAMO', mint: '7xKXtg2CW87d97TXJSDpbD5jBkheTqA83TZRuJosgAsU' },
+  { token: 'bSOL', mint: 'bSo13r4TkiE4KumL71LsHTPpL2euBYLFx6h9HP3piy1' },
+  { token: 'LaineSOL', mint: 'LAinEtNLgpmCP9Rvsf5Hn8W6EhNiKLZQti1xfWMLy6X' },
+  { token: 'HADES', mint: 'BWXrrYFhT7bMHmNBFoQFWdsSgA3yXoAnMhDK6Fn1eSEn' },
```

Comment:
please update the docs for creating new strats and adding new tokens to keep this one up to date as well

Reply:
Will update and add to one of my PRs that I have in historian, thanks for the reminder